### DISCUSSION TITLE
Lost access to deployed repositories through GitHub

### DISCUSSION THREAD

**Discussion by saltoriousSIG** [vercel/community#3085] [Bug Report]
I recently started a  new job and linked my GitHub account to the new team vercel. I previously had a vercel account linked to this GitHub where I had a few different repositories deployed, a few domains purchased, and where I was an admin of a team. Now when I login with my GitHub account I only see the new team I've joined. When I try to login with the email connected to my GitHub (same email deployment notices go to) I get an account not found error. 

How can I recover access to my deployed repositories?

**Comment by amyegan** [comment]
Sorry you're having trouble with your accounts, @saltoriousSIG. Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team. That will let us discuss your account information safely out of this public post 🙏

**Comment by raveauigotzb** [comment]
i have the same trouble(
